.gitignore

```diff
@@ -3,3 +3,4 @@ target/
 .classpath
 .project
 .theia
+my-key.json
```

Comment:
You can always move this out of the git repo and set the env variable accordingly. This is fine too, as long as we don't check them in.

Reply:
Yeah I had it that way before, but I got the error that was solved by restarting the VM and so I thought I might've needed to move it in since it said "portfolio".